src/views/Factboxes/index.tsx

```diff
@@ -245,13 +259,7 @@ export const Factboxes = (): JSX.Element => {
               })()
             }}
           >
-            <SearchInput
-              className='p-4 w-full text-sm border-none focus:border-none'
-              type='text'
-              placeholder='Sök faktaruta'
-              name='factboxes'
-              ref={inputRef}
-            />
+            <Input placeholder='Sök faktaruta' ref={inputRef} name='factboxes' autoFocus />
```

Comment:
I think we need to cancel the focus:ring on the input

Reply:
Should it be done separately here? Preferably in `elephant-ui`, no?

Reply:
We'll leave it as is as the plan probably is to have the Search view for searching factboxes as well.